src/components/Cards/TaskCard/TaskCard.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import type { Meta, StoryFn, StoryObj } from '@storybook/nextjs-vite';
+import { TaskCard } from './TaskCard';
+import { Badge } from 'src/components/Badge/Badge';
+import CheckIcon from '@mui/icons-material/Check';
+import RefreshIcon from '@mui/icons-material/Refresh';
+import React from 'react';
+import Stack from '@mui/material/Stack';
+
+const meta: Meta<typeof TaskCard> = {
+  title: 'Components/Cards/Mission task cards',
+  component: TaskCard,
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof TaskCard>;
+
+const Template: StoryFn<typeof TaskCard> = (args) => <TaskCard {...args} />;
+
+export const Default: Story = {
+  render: Template,
+  args: {
+    title: 'Default task',
+    description: 'This is a sample task description.',
+  },
+};
+
+export const Active: Story = {
+  render: Template,
+  args: {
+    title: 'Active Task',
+    description: 'This task is currently active.',
+    isActive: true,
+  },
+};
+
+export const Loading: Story = {
+  render: Template,
+  args: {
+    isLoading: true,
+  },
+};
+
+export const WithVerifiedStatus: Story = {
+  render: Template,
+  args: {
+    title: 'Task with verified status',
+    description: 'You have completed this task successfully.',
+    statusBadge: (
+      <Badge label="Verified" variant="success" startIcon={<CheckIcon />} />
+    ),
+  },
+};
+
+export const WithVerifyingStatus: Story = {
+  render: Template,
+  args: {
+    title: 'Verifying Task',
+    description: 'Please wait while we verify your progress.',
+    statusBadge: (
+      <Badge
+        label="Verify"
+        variant="disabled"
+        startIcon={
+          <RefreshIcon
+            sx={{
+              animation: 'spin 1s linear infinite',
+              '@keyframes spin': {
+                from: { transform: 'rotate(0deg)' },
+                to: { transform: 'rotate(360deg)' },
+              },
+            }}
+          />
+        }
+      />
+    ),
+  },
+};
+
+export const WithTaskTypes: Story = {
+  render: () => (
+    <Stack spacing={2}>
+      <TaskCard
+        title="Bridge ETH"
```

Comment:
Would be good to use translation

Reply:
Do we want to use translations for the stories @oktapodia?